pages/api/extension/trigger.ts

```diff
@@ -0,0 +1,48 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { getToken } from 'next-auth/jwt'
+
+export default async function triggeHandler(req: NextApiRequest, res: NextApiResponse) {
+	// For cors prefetch options request
+	if (req.method == "OPTIONS") {
+		res.setHeader("Access-Control-Allow-Headers", "Content-Type, Authorization");
+		res.setHeader("Access-Control-Allow-Origin", "*");
```

Comment:
You can remove these two lines because of #291 
(Also, can you also test if the API works properly without the entire condition checking for OPTIONS method? The middleware.ts's addition might have completely eliminated the problem)

Reply:
done